### ISSUE TITLE
Avast prevent for XCI-cutter execution

### ISSUE THREAD

**Issue by DavidBabel** [Destiny1984/XCI-Cutter#4]
Hey hi !

For now, this tool is sended is safezone by any antivirus software. What part of the code can do that ?
this only appends on version 2.0, recently released.

**Comment by Ceraus** [comment]
That's odd! 2.0 works/open perfectly fine on Avast Program version 18.4.2338 / Virus definitions version: 180527-2; Which version are you on? And also, Virustotal scans it as completely clear as well. So I am unsure what the underlying issue may be?

https://www.virustotal.com/#/file/3dddf97b285e370b16a1877e18a5ab7bbaf23a17695288ebc1e85a9ef08af80f/detection

**Comment by Destiny1984** [comment]
@DavidBabel 
What exactly does it report?

There seems to be a problem with one specific avast version. 
Maybe it doesn't like how i implemented the automatic deletion of input files?

All in all, it's a false alarm.

**Comment by DavidBabel** [comment]
It works today, but for hours yesterday i was blocked in a state saying "your file was sended to avast, we test the program and it begin to crack. We will have finished in around 120 minutes".

And i was not able to allow the program to run at all. I tried 2 computer with the same issue yesterday. I just tested again and both are ok now. I tried on a third, and it worked as expected.

Maybe i downloaded the file as a first avast owner
